app/src/main/java/com/worldsnas/starplayer/di/modules/DatabaseModule.kt

```diff
@@ -0,0 +1,21 @@
+package com.worldsnas.starplayer.di.modules
+
+import android.app.Application
+import androidx.room.Room
+import com.worldsnas.starplayer.model.persistent.AppDataBase
+import dagger.Module
+import dagger.Provides
+
+@Module
+object DatabaseModule {
+
+    @Provides
+    @JvmStatic
```

Comment:
benazaram behtare `@singleton` bashe

Reply:
nemishe scope jodaee bezaram az componenti ke dakhelesh hast

Reply:
age eshtebah nakonam to AppComponent e
va scope on component @Singleton e
bara in goftam

Reply:
doroste vali khata mide bedalil motefavet bodan scope, faqat scope hamin component ro mishe behesh dad